### PR TITLE
Minor Refactor

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
# Changes

- isort imports
- Fix OpenVINO naming
- Remove inference parameters from `template.yaml`
# Coverage

image

### CODE REVIEW

File: external/anomalib/anomaly_classification/openvino.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 from typing import Optional, Union
 
 from addict import Dict as ADDict
+from anomalib.core.model.inference import OpenVinoInferencer
```
Comment (samet-akcay) [inline external/anomalib/anomaly_classification/openvino.py:25]: Should we rename this in `anomalib` to match the new naming convention? Maybe  I could quickly add this to my existing config [MR](https://gitlab-icv.inn.intel.com/algo_rnd_team/anomaly/-/merge_requests/143)

Comment (ashwinvaidya17) [reply external/anomalib/anomaly_classification/openvino.py:25]: That would be nice. Plus I guess this convention might also be something to do with the trademark.

Comment (samet-akcay) [reply external/anomalib/anomaly_classification/openvino.py:25]: in terms of trademark, I agree. It is just against our camel case naming convention, but should be alright

Comment (ashwinvaidya17) [reply external/anomalib/anomaly_classification/openvino.py:25]: Yeah I agree. It breaks our coding convention.

**Review by djdameln — APPROVED** [review]
Looks good to me!

**Review by samet-akcay — APPROVED** [review]

**Pull request merged**